src/indexable.cr

```diff
@@ -412,24 +412,80 @@ module Indexable(T)
     indexes.map { |index| self[index] }
   end
 
+  # Calls the given block once for each index in `self`, passing `self`
+  # and `other` index element as parameter.
+  #
+  # ```
+  # a = [1, 2, 3]
+  # b = ["a", "b", "c"]
+  #
+  # a.zip(b)  { |x,y| puts "[ #{x} -- #{y} ]" }
+  # ```
+  #
+  # produces:
```

Comment:
The example would be more condensed if t was in one code block and the printed output just commented lines.

Reply:
It is possible, I firstly did like you mentioned, but after some research, I followed the example what exists at [`src/dir.cr`](https://github.com/crystal-lang/crystal/blob/master/src/dir.cr#L38-L55) to keep the consistency.

Reply:
I'm not sure consistency is that important for this and besides, we can change the example for `Dir` as well...
It's just an unnecessary division between the example code and output. IMHO it should be as condensed as possible For examples that cover multiple files, I usually just print them in one code block and use comments to indicate different files. This keeps things compact and connected.